### PR TITLE
[T3A3][T16-A2]Xie Jihui

### PR DESCRIPTION
**Pull request by xjh666**

### CODE REVIEW

**Comment by xjh666** [comment]
Ready for review

File: src/seedu/addressbook/data/address/info/Block.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+public class Block {
+	private String block;
+	
+	public Block(){
+		block = null;
+	}
+	
+	public void addInfo(String block){
+		this.block = block;
+	}
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/address/info/Block.java]: The setting of this variable could be done in the constructor, reducing the need to call both the constructor and this setter.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -11,10 +11,14 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    
+    public Block block = new Block();
+    public PostalCode postalCode = new PostalCode();
+    public Street street = new Street();
+    public Unit unit = new Unit();
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/Address.java]: Should these variables be public?

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -26,6 +30,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        String []info = address.split(", ");
+        this.block.addInfo(info[0]);
+        this.street.addInfo(info[1]);
+        this.unit.addInfo(info[2]);
+        //this.postalCode.addInfo(Integer.parseInt(info[3]));
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/Address.java]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

**Comment by PierceAndy** [comment]
Avoid making commits with empty commit messages.

@xjh666 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**